packages/sources/src/addSourceWizard/hardcodedComponents/openshift/costManagement.js

```diff
@@ -35,10 +35,10 @@ export const ConfigureCostOperator = () => {
             <Text>
                 { intl.formatMessage({
                     id: 'cost.openshift.operator_configured',
-                    defaultMessage: 'If you configured the operator to create a source (create_source: true), {stop} here and {cancel} out of this flow.'
+                    defaultMessage: 'If you configured the operator to create a source (create_source: true), <b>STOP</b> here and <b>CANCEL</b> out of this flow.'
```

Comment:
Is this change supposed to be here?

Reply:
yeeeah :sweat_smile: a small refactoring :smirk: